pkg/skuba/actions/node/upgrade/apply.go

```diff
@@ -128,6 +130,29 @@ func Apply(client clientset.Interface, target *deployments.Target) error {
 			return err
 		}
 	}
+
+	/*
+		Always upload crio files, regardless of the version (allows to
+		enforce user behavior during patch updates).
+		During the upgrade from 1.16 to 1.18 crio, the cri-o package will
+		handle overriding the old crio sysconfig to an "empty" sysconfig,
+		and the cri.configure action will be enough.
+		We can remove the conditionals and only
+		keep the cri.configure action when caasp 4.2.0 is not supported
+		anymore (everyone has updated crio to 1.18)
+	*/
```

Comment:
nit: could you please make the comments shift a tab space?

Reply:
Not sure what you meant here? It's my IDE that did the shifting :p 
Do you mean indent with one more tab?

Reply:
> Do you mean indent with one more tab?

yes

Reply:
I just tried with one more and one less tab for the indented text, and my IDE is always overriding. Probably due to the autoformatting config I have.

It doesn't complain on
```
	/*
	 *	Always upload crio files, regardless of the version (allows to
	 *	enforce user behavior during patch updates).
	 *	During the upgrade from 1.16 to 1.18 crio, the cri-o package will
	 *	handle overriding the old crio sysconfig to an "empty" sysconfig,
	 *	and the cri.configure action will be enough.
	 *	We can remove the conditionals and only
	 *	keep the cri.configure action when caasp 4.2.0 is not supported
	 *	anymore (everyone has updated crio to 1.18)
	 */
```

Reply:
I can also do  ``// `` on everyline. It doesn't matter to me :)

Reply:
I personally prefer `//` on every line.

Reply:
Will do if we need another patch and another person wants this.
No point for restarting the whole CI machine for a nit I guess...